### ISSUE TITLE
sidebar current page item should not hide episode number

### ISSUE THREAD

**Issue by zkamvar** [carpentries/sandpaper#432]
from https://github.com/carpentries/workbench/issues/42:

> When I click on the menu let's say number 2 in a list. It changes to a bullet point so you don't know where you are. It should stay as a number 2. At first I thought it took me to a level down and took forever long for me to figure out it was not moving between levels just changing the number to a bullet. It is the worst and makes you loose yourself in the navigation. It is confusing to new users as well. You have a hard time directing people to the links - now I hoover and point out the change and have to click several times back and forth for people to follow in a lesson.
>
> --- @cynthiaftw 

This should be fixed here by modifying how we establish the title. At the moment, we assume that if we are updating a sidebar element, then the title does not need a number, but we can recycle the title that is currently in the sidebar for this. 

https://github.com/carpentries/sandpaper/blob/870fe04171131f69b0f1cb6b718dd6f90798b360/R/utils-sidebar.R#L132-L149
